index.html

```diff
@@ -23,14 +23,60 @@ <h1>CASTING RECALL</h1>
         </form>
       </div>
     </div>
-    <div class="row j-center">
-      <h5>Search for a film to recast the characters!</h5>
+    <div data-view="landing" class="">
+      <div class="row j-center">
+        <div>
+          <h5>Search for a film to recast the characters!</h5>
+        </div>
+      </div>
+    </div>
+    <div data-view="search" class="hidden">
+      <div class="row j-center">
+        <h5>Search for a film to recast the characters!</h5>
+      </div>
+      <div>
+        <ul class="ul column-full" id="render-movie-list"></ul>
+      </div>
+    </div>
+    <div data-view="cast" class="hidden">
+      <div class="row wrap j-b">
+        <div>
+          <h5>Cast:</h5>
+        </div>
+        <div class="nav a-center">
+          <a class="w-t hidden">save cast</a>
+          <a id="cast-go-b" class="">go back</a>
+        </div>
+      </div>
+      <div>
+        <ul class="ul column-full" id="render-cast-list"></ul>
+      </div>
+    </div>
+    <div data-view="people" class="hidden">
+      <div class="row j-center j-b">
+        <div>
+          <h5 id="actor-to-replace">Replace actor to replace with:</h5>
+        </div>
+        <div class="a-center">
+          <a id="people-go-b" class="">go back</a>
+        </div>
+      </div>
+      <div id="people-search-div" data-view="people" class="a-center column-full hidden">
+        <form id="people-search" class="row">
+          <input class="input" id="people-search-input" type="text" placeholder="Search for an actor to swap" required>
+        </form>
+      </div>
+      <div>
+        <ul class="ul column-full" id="render-people-list"></ul>
+      </div>
     </div>
     <div>
-      <ul class="ul column-full" id="render-list">
-      </ul>
+      <!-- <ul class="ul column-full" id="render-movie-list"></ul> -->
+      <!-- <ul class="ul column-full" id="render-cast-list"></ul> -->
+      <!-- <ul class="ul column-full" id="render-people-list"></ul> -->
     </div>
   </main>
+  <script src="https://kit.fontawesome.com/878479db5c.js" crossorigin="anonymous"></script>
```

Comment:
Not needed. Font awesome was already included for you in the `head` of your HTML